packages/core/strapi/tests/search.test.e2e.js

```diff
@@ -330,14 +287,6 @@ describe('Search query', () => {
 
       expect(res.body.data.length).toBe(1);
       expect(res.body.data[0]).toMatchObject(transformToRESTResource(data.bed[0]));
-      expect(res.body.meta).toMatchObject({
-        pagination: {
-          page: 1,
-          pageSize: 10,
-          pageCount: 1,
-          total: 1,
-        },
-      });
```

Comment:
Why did you delete this part?

Reply:
I added it by mistake when I was fixing the search test. it didn't make much sense to check the pagination on the search test we should have dedicated tests :)